source/core-concepts/deployments.rst

```diff
@@ -144,3 +158,30 @@ You can also create a new deployment version using the Valohai APIs
     `Valohai API for Automation <https://docs.valohai.com/valohai-api/>`_
 
 ..
+
+
+Deployment monitoring
+~~~~~~~~~~~~~~~~~~~~~~
+
+Under each deployment version, you can view the deployment logs from your deployment endpoints.
+
+You can collect additional metrics from your deployments by printing JSON from your deployment endpoint. Valohai will collect these just like execution metrics, and allow you to chart them in both time series and histogram modes.
+
```

Comment:
Maybe an admonition about needing to wrap the data in `vh_metadata` would be a good idea, considering the above paragraph says "just like" :)

Reply:
good idea! Fixed